spa/public/index.html

```diff
@@ -8,6 +8,15 @@
     <meta name="theme-color" content="#000000" />
     <meta name="description" content="RevEngine" />
 
+    <!-- Google Tag Manager -->
+    <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
```

Comment:
This script must be below the "csp-nonce" meta tag on line 20.  Also, we need to add the nonce as an attribute to the script tag, as below:
```suggestion
    <script nonce="{{ request.csp_nonce }}">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
```

Reply:
Should this be inside the conditional like the other script tag?